### ISSUE TITLE
API v20 has undocumented and weird methods naming change in AdAccount

### ISSUE THREAD

**Issue by avmi** [facebook/facebook-java-business-sdk#465]
### Which SDK version are you using?
20.0.0

### What's the issue?
Undocumented and strange looking method rename caused build to fail, because we're using `AdAccount.getTargetingValidation`.

### Steps/Sample code to reproduce the issue

```
  public static class APIRequestCreateMAnAgeDPartnerAd extends APIRequest<APINode> {
```

https://github.com/facebook/facebook-java-business-sdk/blob/be6bc7b70d35b5dba61476511d91d2c743309cbd/src/main/java/com/facebook/ads/sdk/AdAccount.java#L21810

```
  public APIRequestGetTargetingValidATIOn getTargetingValidATIOn() {
    return new APIRequestGetTargetingValidATIOn(this.getPrefixedId().toString(), context);
  }
```

https://github.com/facebook/facebook-java-business-sdk/blob/be6bc7b70d35b5dba61476511d91d2c743309cbd/src/main/java/com/facebook/ads/sdk/AdAccount.java#L791

and couple more occurences in the same file.

#### Observed Results:

Causes failing build. Solved by rolling back to 19.0.3.

#### Expected Results:

Names should stay the same as in 19.0.3. Release notes doesn't mention any changes like this

**Comment by YaroslavSahach** [comment]
Have the same issue with `Post.getShareDPosts()`. Before this method was called `getSharedPosts()`.

**Comment by stcheng** [comment]
thank you for reporting this. we're aware of this and are actively working on a solution/

**Comment by stcheng** [comment]
this issue has been addressed in 6d1d4a2fea944b368e125801a492d08c836c63b9 and we're to release a new version
